app/controllers/admin/schedules_controller.rb

```diff
@@ -28,6 +28,25 @@ def index
       end
     end
 
+    def create
+      invoke_callbacks(:create, :before)
```

Comment:
can you please delete this callback code? I am pretty sure this is not used.
3 less lines of code in this method.